### ISSUE TITLE
Test failure in Alpine based Docker container

### ISSUE THREAD

**Issue by szabgab** [cho45/Test-Time#9]
```
cpanm (App::cpanminus) 1.7044 on perl 5.026002 built for x86_64-linux-thread-multi
Work directory is /root/.cpanm/work/1531049720.21520
You have make /usr/bin/make
You have /usr/bin/wget
You have /bin/tar: tar (busybox) 1.27.2
You have /usr/bin/unzip
Searching Test::Time () on cpanmetadb ...
--> Working on Test::Time
Fetching http://www.cpan.org/authors/id/S/SA/SATOH/Test-Time-0.06.tar.gz
-> OK
Unpacking Test-Time-0.06.tar.gz
Entering Test-Time-0.06
Checking configure dependencies from META.json
Checking if you have ExtUtils::MakeMaker 6.64 ... Yes (7.24)
Configuring Test-Time-0.06
Running Makefile.PL
Checking if your kit is complete...
Looks good
Generating a Unix-style Makefile
Writing Makefile for Test::Time
Writing MYMETA.yml and MYMETA.json
-> OK
Checking dependencies from MYMETA.json ...
Checking if you have Test::More 0 ... Yes (1.302073)
Building and testing Test-Time-0.06
cp lib/Test/Time.pm blib/lib/Test/Time.pm
Manifying 1 pod document
PERL_DL_NONLAZY=1 "/usr/bin/perl" "-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef *Test::Harness::Switches; test_harness(0, 'bli
b/lib', 'blib/arch')" t/*.t
t/00_compile.t .. ok

#   Failed test 'localtime() in scalar context correct'
#   at t/01_base.t line 17.
#          got: 'Thu Jan  1 00:00:02 1970'
#     expected: 'Thu Jan  1 01:00:02 1970'

#   Failed test 'localtime() in list context correct'
#   at t/01_base.t line 21.
#     Structures begin differing at:
#          $got->[2] = '0'
#     $expected->[2] = '1'

#   Failed test 'localtime() in scalar context with argument correct'
#   at t/01_base.t line 24.
#          got: 'Thu Jan  1 00:01:40 1970'
#     expected: 'Thu Jan  1 01:01:40 1970'

#   Failed test 'localtime() in list context with argument correct'
#   at t/01_base.t line 28.
#     Structures begin differing at:
#          $got->[2] = '0'
#     $expected->[2] = '1'
# Looks like you failed 4 tests of 9.
t/01_base.t .....
Dubious, test returned 4 (wstat 1024, 0x400)
Failed 4/9 subtests


Test Summary Report
-------------------
t/01_base.t   (Wstat: 1024 Tests: 9 Failed: 4)
  Failed tests:  4-7
  Non-zero exit status: 4
Files=2, Tests=10,  1 wallclock secs ( 0.02 usr  0.02 sys +  0.19 cusr  0.03 csys =  0.26 CPU)
Result: FAIL
Failed 1/2 test programs. 4/10 subtests failed.
make: *** [Makefile:840: test_dynamic] Error 4
-> FAIL Installing Test::Time failed. See /root/.cpanm/work/1531049720.21520/build.log for details. Retry with --force to force install
it.
```

Using Docker image built with this Dockerfile on OSX:

```
FROM alpine:3.7
RUN apk update && \
    apk add wget && \
    apk add perl && \
    apk add perl-dev && \
    apk add make && \
    apk add g++ && \
    wget https://cpanmin.us/ && \
    perl index.html App::cpanminus && \
    rm index.html

WORKDIR /root
```

and then running

```
cpanm Test::Time
```

**Comment by kb1** [comment]
Try adding "apk add tzdata"

**Comment by cho45** [comment]
Fixed on #13 and released as 0.08.
